engine/source/engine.cpp

```diff
@@ -248,7 +249,21 @@ void OldEngine::Tick(Engine& engine)
 
     if (inputDeviceManager.IsKeyPressed(KeyboardCode::eF1))
     {
-        rendererModule.GetRenderer()->GetDebugPipeline().SetState(!rendererModule.GetRenderer()->GetDebugPipeline().GetState());
+        physicsModule.debugRenderer->SetState(!physicsModule.debugRenderer->GetState());
+    }
+
+    if (inputDeviceManager.IsKeyPressed(KeyboardCode::eF2))
+    {
+        pathfindingModule.SetDebugDrawState(!pathfindingModule.GetDebugDrawState());
+    }
+
+    if (physicsModule.debugRenderer->GetState() || pathfindingModule.GetDebugDrawState())
```

Comment:
ah nice, good for now. Later when we update the debug draw we can add enum flags for this